client/templates/tweet/tweetAdminControlsSmall.html

```diff
@@ -1,4 +1,4 @@
-<md-menu ng-style="setAdminButtonSize()">
+<md-menu md-offset="0 50" ng-style="setAdminButtonSize()">
```

Comment:
If you add md-position-mode="target-right target" it sort of right aligns and looks a bit more natural.